.github/workflows/blackduck.yml

```diff
@@ -12,14 +12,13 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
```

Comment:
[req] I guess we don't need the matrix here anymore.

Reply:
=> `runs-on: ubuntu-latest`

Reply:
We use the matrix below as a variable for the name. This could be hardcoded as well, but I wasn't sure if this would break something.

Reply:
If it does we can fix it ;)